yarn.lock

```diff
@@ -2188,13 +2188,20 @@ aws4@^1.8.0:
   resolved "https://registry.yarnpkg.com/aws4/-/aws4-1.10.0.tgz#a17b3a8ea811060e74d47d306122400ad4497ae2"
   integrity sha512-3YDiu347mtVtjpyV3u5kVqQLP242c06zwDOgpeRnybmXlYYsLbtTrUBUm8i8srONt+FWobl5aibnU1030PeeuA==
 
-axios@^0.19.1, axios@^0.19.2:
+axios@^0.19.1:
   version "0.19.2"
   resolved "https://registry.yarnpkg.com/axios/-/axios-0.19.2.tgz#3ea36c5d8818d0d5f8a8a97a6d36b86cdc00cb27"
   integrity sha512-fjgm5MvRHLhx+osE2xoekY70AhARk3a6hkN+3Io1jc00jtquGvxYlKlsFUhmUET0V5te6CcZI7lcv2Ym61mjHA==
   dependencies:
     follow-redirects "1.5.10"
 
+axios@^0.21.1:
+  version "0.21.1"
+  resolved "https://registry.yarnpkg.com/axios/-/axios-0.21.1.tgz#22563481962f4d6bde9a76d516ef0e5d3c09b2b8"
+  integrity sha512-dKQiRHxGD9PPRIUNIWvZhPTPpl1rf/OxTYKsqKUDjBwYylTvV7SjSHJb9ratfyzM6wCdLCOYLzs73qpg5c4iGA==
+  dependencies:
+    follow-redirects "^1.10.0"
+
```

Comment:
This kind of thing concerns me because it means that there may be two modules in memory at the same time, so that when module M in a library references, say, Axios thing X(0.19.1), and module N in our code references X(0.21.1), then a test like `X(0.19.1) instanceof X(0.21.1)` produces `false`. This bit me in the early days of my tenure at LifeOmic, and I've seen error handling code in recent PRs that use `instanceof` for (I think it was) Axios errors. Perhaps I'm superstitious now? 🤷‍♂️ 

@mknoedel @ndowmon @austinkelleher I guess we go ahead and merge this and not worry about that, but thought I'd mention it.

Reply:
Here is the output of `yarn why axios` from `master` and then from this branch:

```
j1/gh/sdk  master ✔                                                                                                                                                                           7h21m  
▶ yarn why axios
yarn why v1.22.4
[1/4] 🤔  Why do we have the module "axios"...?
[2/4] 🚚  Initialising dependency graph...
[3/4] 🔍  Finding dependency...
[4/4] 🚡  Calculating file sizes...
=> Found "axios@0.19.2"
info Reasons this module exists
   - "_project_#@jupiterone#integration-sdk-runtime" depends on it
   - Hoisted from "_project_#@jupiterone#integration-sdk-runtime#axios"
   - Hoisted from "_project_#@jupiterone#integration-sdk-runtime#@lifeomic#alpha#axios"
info Disk size without dependencies: "480KB"
info Disk size with unique dependencies: "648KB"
info Disk size with transitive dependencies: "776KB"
info Number of shared dependencies: 3
✨  Done in 0.59s.

j1/gh/sdk  master ✔                                                                                                                                                                           7h21m  
▶ git co dependabot/npm_and_yarn/axios-0.21.1
Branch 'dependabot/npm_and_yarn/axios-0.21.1' set up to track remote branch 'dependabot/npm_and_yarn/axios-0.21.1' from 'origin'.
Switched to a new branch 'dependabot/npm_and_yarn/axios-0.21.1'

j1/gh/sdk  dependabot/npm_and_yarn/axios-0.21.1 ✔                                                                                                                                             7h20m  
▶ yarn
yarn install v1.22.4
[1/4] 🔍  Resolving packages...
[2/4] 🚚  Fetching packages...
[3/4] 🔗  Linking dependencies...
[4/4] 🔨  Building fresh packages...
✨  Done in 2.91s.

j1/gh/sdk  dependabot/npm_and_yarn/axios-0.21.1 ✔                                                                                                                                             7h20m  
▶ yarn why axios                             
yarn why v1.22.4
[1/4] 🤔  Why do we have the module "axios"...?
[2/4] 🚚  Initialising dependency graph...
[3/4] 🔍  Finding dependency...
[4/4] 🚡  Calculating file sizes...
=> Found "axios@0.21.1"
info Reasons this module exists
   - "_project_#@jupiterone#integration-sdk-runtime" depends on it
   - Hoisted from "_project_#@jupiterone#integration-sdk-runtime#axios"
info Disk size without dependencies: "504KB"
info Disk size with unique dependencies: "548KB"
info Disk size with transitive dependencies: "548KB"
info Number of shared dependencies: 1
=> Found "@lifeomic/alpha#axios@0.19.2"
info This module exists because "_project_#@jupiterone#integration-sdk-runtime#@lifeomic#alpha" depends on it.
info Disk size without dependencies: "480KB"
info Disk size with unique dependencies: "524KB"
info Disk size with transitive dependencies: "524KB"
info Number of shared dependencies: 1
✨  Done in 0.57s.
```

Note that `@jupiterone/integration-sdk-runtime` is going to be served runtime modules from `0.21.1` while `@lifeomic/alpha` is going to be using them from `0.19.2`. This isn't a problem as long as there is never a need to do module identity comparisons in runtime code that will see "the same type" from two different modules. For example, errors raised with a module/type from the latter will not be `instanceof` the error types referenced in the former.

Has anyone else run into this madness IRL?

Reply:
Can you also bump `@lifeomic/alpha`? It's currently using `@^0.21.1`. https://github.com/lifeomic/alpha/blob/master/package.json

Reply:
That would resolve the multiple versions issue, I believe.